resources/analysisTools/rnaseqworkflow/HIPO2_rnaseq_processing.sh

```diff
@@ -262,11 +262,20 @@ if [ "$RUN_ARRIBA" == true ]
 then
 	make_directory $ARRIBA_DIR
 	cd $ARRIBA_DIR
-	echo_run "$ARRIBA_READTHROUGH_BINARY -g $GENE_MODELS -i $ALIGNMENT_DIR/$STAR_SORTED_MKDUP_BAM -o ${SAMPLE}_${pid}_merged_read_through.bam"
-	echo_run "$ARRIBA_BINARY -c $ALIGNMENT_DIR/$STAR_CHIMERA_MKDUP_BAM -r ${SAMPLE}_${pid}_merged_read_through.bam -x $ALIGNMENT_DIR/$STAR_SORTED_MKDUP_BAM -a $GENOME_FA -k $ARRIBA_KNOWN_FUSIONS -g $GENE_MODELS -b $ARRIBA_BLACKLIST -o ${SAMPLE}_${pid}.fusions.txt -O ${SAMPLE}_${pid}.discarded_fusions.txt "
-	if [[ -f "${SAMPLE}_${pid}.fusions.txt" ]]
+
+	arribaCommand="$ARRIBA_BINARY -c '$ALIGNMENT_DIR/$STAR_CHIMERA_MKDUP_BAM' -x '$ALIGNMENT_DIR/$STAR_SORTED_MKDUP_BAM' -a '$GENOME_FA' -k '$ARRIBA_KNOWN_FUSIONS' -g '$GENE_MODELS' -b '$ARRIBA_BLACKLIST' -T -P -o '${SAMPLE}_$pid.fusions.txt' -O '${SAMPLE}_$pid.discarded_fusions.txt'"
+
+    if [ "$useSingleEndProcessing" == true ]; then
+        arribaCommand="$arribaCommand -s yes"
+    else
+        arribaCommand="$arribaCommand -s no"
+    fi
+
+    echo_run "$arribaCommand"
+
+	if [[ -f "${SAMPLE}_$pid.fusions.txt" ]]
 	then
-		echo_run "$ARRIBA_DRAW_FUSIONS --annotation=$GENE_MODELS --fusions=${SAMPLE}_${pid}.fusions.txt --output=${SAMPLE}_${pid}.fusions.pdf"
+		echo_run "$ARRIBA_DRAW_FUSIONS --annotation='$GENE_MODELS' --fusions='${SAMPLE}_$pid.fusions.txt' --output='${SAMPLE}_$pid.fusions.pdf'"
```

Comment:
The new `draw_fusions.R` can do much more than the old version. But it needs some additional files for that. Here is the enhanced command:

```
echo_run "$ARRIBA_DRAW_FUSIONS --annotation='$GENE_MODELS' --proteinDomains='$ARRIBA_PROTEIN_DOMAINS' --cytobands='$ARRIBA_CYTOBANDS' --alignments='$ALIGNMENT_DIR/$STAR_SORTED_MKDUP_BAM' --fusions='${SAMPLE}_$pid.fusions.txt' --output='${SAMPLE}_$pid.fusions.pdf'"
```

`$ARRIBA_PROTEIN_DOMAINS` should be set to `/path/to/protein_domains_hg19_hs37d5_GRCh37_2019-07-05.gff3` and `$CYTOBANDS` to `/path/to/cytobands_hg19_hs37d5_GRCh37_2018-02-23.tsv`. The files can be found in the [Arriba release v1.2.0](https://github.com/suhrig/arriba/releases/download/v1.2.0/arriba_v1.2.0.tar.gz) in the `database` folder.

Reply:
Many thanks for these suggestions!